package.json

```diff
@@ -81,6 +81,11 @@
                 "category": "Bazel",
                 "command": "bazel.showExtensionLog",
                 "title": "Show Extension Log"
+            },
+            {
+                "category": "Bazel",
+                "command": "bazel.copyTargetToClipboard",
+                "title": "Copy to Clipboard"
```

Comment:
Let's call this "Copy Label to Clipboard" to make it clear what's being copied (the [label](https://docs.bazel.build/versions/master/build-ref.html#labels) as opposed to, say, file contents of some sort).

Reply:
Ah thanks, I did not know the //path was called label.